Leanplum-SDK/Classes/Features/Actions/LPActionContext.m

```diff
@@ -188,7 +188,10 @@ - (void)setProperArgs
         if (parent) {
             _args = [parent getChildArgs:_key];
         } else if (_messageId) {
-            _args = [LPVarCache sharedCache].messages[_messageId][LP_KEY_VARS];
+            NSDictionary *cachedArgs = [LPVarCache sharedCache].messages[_messageId][LP_KEY_VARS];
+            if (cachedArgs) {
+                _args = cachedArgs;
+            }
```

Comment:
Still coudnt understand, here, may be we can do a quick pair demo for this one.
Also, meanwhile, can you please look into adding test case for this change?